### PR TITLE
Register Cesium C compiler

### PR DESCRIPTION
**Pull request by kant2002**
Add Cesium C compiler which compiles C to IL

### CODE REVIEW

**Review by kant2002 — COMMENTED** [review]
Change tags

**Comment by kant2002** [comment]
@shiftkey would it be appropriate to ping you, so you can review PR ?

File: _data/projects/Cesium.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: Cesium
+desc: C compiler for the CLI platform
+site: https://github.com/fornever/cesium
+tags:
+  - .net
+  - compiler
+  - с
+  - с#
```
Comment (ritwik12) [inline _data/projects/Cesium.yml]:
```suggestion
- .net
- compiler
- с
- с#
```

**Review by ritwik12 — DISMISSED** [review]

**Comment by kant2002** [comment]
@ritwik12 can you please review this again?

**Comment by shiftkey** [comment]
@kant2002 the reason why this PR was stuck is that the validation wanted an ASCII C character, but your file had some unrecognized characters (this `D181`) sequence:

Screenshot 2023-08-27 at 3 23 01 PM

I noticed in VSCode that there was also a warning here about this:


Screenshot 2023-08-27 at 3 26 54 PM

File: _data/projects/Cesium.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+name: Cesium
+desc: C compiler for the CLI platform
+site: https://github.com/fornever/cesium
+tags:
+- .net
+- compiler
+- c
+- c#
+upforgrabs:
+  name: up for grabs
```
Comment (shiftkey) [inline _data/projects/Cesium.yml]: :art: label must match the value used

```suggestion
  name: up-for-grabs
```

File: _data/projects/Cesium.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+name: Cesium
+desc: C compiler for the CLI platform
+site: https://github.com/fornever/cesium
+tags:
+- .net
+- compiler
+- c
+- c#
+upforgrabs:
+  name: up-for-grabs
```
Comment (shiftkey) [inline _data/projects/Cesium.yml]:
```suggestion
  name: good-first-issue
```

**Review by shiftkey — APPROVED** [review]

**Pull request merged**